src/libraries/System.IO.Pipelines/ref/System.IO.Pipelines.cs

```diff
@@ -54,6 +54,8 @@ public abstract partial class PipeReader
         [System.ObsoleteAttribute("OnWriterCompleted may not be invoked on all implementations of PipeReader. This will be removed in a future release.")]
         public virtual void OnWriterCompleted(System.Action<System.Exception?, object?> callback, object? state) { }
         public abstract System.Threading.Tasks.ValueTask<System.IO.Pipelines.ReadResult> ReadAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken));
+        public System.Threading.Tasks.ValueTask<System.IO.Pipelines.ReadResult> ReadAtLeastAsync(int minimumBytes, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Probably should have looked more into this during API review, but maybe this should be `minimumSize` instead of `minimumBytes` to align more with the `minimumSegmentSize` param in the PipeOptions ctor. We don't use "bytes" in any of the other param names.

Reply:
@bartonjs sounds good?

Reply:
Well, it's describing the minimum value for ret.Buffer.Length (barring boundary conditions), so `minimumLength` seems better than `minimumSize`.  But I'm currently at my caffeine low point for the day, so take my words with a grain of salt.

@terrajobst, your thoughts?

Reply:
Normally we use bytes rather than size to make it clear that it's number of bytes and not number of elements, but for APIs that deal with spans of byte it doesn't matter (cue The Rock with It doesn't matter). To my ears, size and length is identical in semantics, it's just length makes more sense for APIs that deal with arrays or span because the property is called `Length`.

And lastly aligning it with the other APIs here seems to make the most sense. So I'd pick what has been used before.

Reply:
I'll change it to minimumSize